web/src/lib/utils/helpers.ts

```diff
@@ -144,6 +144,18 @@ export const getUserNameOrEmail = (userData: UserData | undefined): string => {
   }
 };
 
+export const getNavBarBusinessTitle = (userData: UserData | undefined): string => {
+  const Config = getMergedConfig();
+  if (userData?.profileData.industryId === undefined || userData.profileData.legalStructureId === undefined) {
+    return Config.navigationDefaults.navBarGuestText;
+  }
+  if (userData?.profileData.businessName === undefined || userData?.profileData.businessName === "") {
```

Comment:
This can be simplified to `!userData?.profileData.businessName` because both undefined and empty string are falsey.

Same with above, `=== undefined` can be replaced with a `!` instead

Reply:
Indeed.